corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,27 +34,32 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
     <div data-bind="saveButton: saveButton"></div>
     <div class="row">
         <div class="col-md-10">
-            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">Export to Excel</a>
-            <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">Import from Excel</a>
+            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">{% trans "Export to Excel" %}</a>
```

Comment:
More minor ui feedback:
Minor UI feedback in this area:
- Move the case type to the very top (because it's effectively the title of the page)
- Move the deprecation toggle over to the right and make it the default grey instead of bluish (because it's fairly unimportant)
- Would an expand/collapse all groups be useful?